src/shadowbox/model/errors.ts

```diff
@@ -32,3 +32,16 @@ export class PortUnavailable extends ShadowboxError {
     super(port.toString());
   }
 }
+
+export class AccessKeyNotFound extends ShadowboxError {
```

Comment:
We should avoid the term `Shadowbox`. It means nothing to the user. `OutlineError` seems more appropriate. Maybe add a TODO?

Reply:
Done.